interfaces/cython/cantera/examples/reactors/ic_engine.py

```diff
@@ -5,6 +5,8 @@
 The simulation uses n-Dodecane as fuel, which is injected close to top dead
 center. Note that this example uses numerous simplifying assumptions and
 thus serves for illustration purposes only.
+
+Requires: Cantera >= 2.5.0, matplotlib >= 2.0
```

Comment:
Can you also document versions for numpy and scipy? Might as well capture all of the dependencies. You can probably just pick some arbitrary recent versions, any features we're using in NumPy and SciPy have been there for a long time and it's probably not worth going all the way back to find the exact version, since people won't even be able to get those old versions most likely.

Reply:
Probably `numpy >= 1.15, scipy >= 0.19` would work.

Reply:
Adding the SciPy version is a good idea. I think numpy can be left as an implicit dependency of Cantera unless the example happens to be using some feature that requires a particularly version.

Reply:
Done.